release-notes/release-2.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: February 13, 2024
+---
+
+# Features
+- PHP Azure Build Pipelines
+- [RUM Mobile - React Native](https://docs.middleware.io/rum/rum-react)
+- Spend Analysis charts
+- Trace correlation drill down from specific log messages
+- Price calculator that analyzes your bill based on the usage of different products
+- Sort and group attributes of log-collected data
+
+# Changes
+- Billing tab now displays information about plan name, billing date, billing address, payment details and option to update billing information
+- Invoice tab now displays all past invoices with invoice date, paid date, and the option to download invoices
```

Comment:
This already exists today, can remove it. We updated the page a little but not much